net-utils/src/ip_echo_server.rs

```diff
@@ -18,6 +19,15 @@ use {
 
 pub type IpEchoServer = Runtime;
 
+// Enforce a minimum of two threads:
+// - One thread to monitor the TcpListener and spawn async tasks
+// - One thread to service the spawned tasks
+pub const MINIMUM_IP_ECHO_SERVER_THREADS: usize = 2;
+// IP echo requests require little computation and come in fairly infrequently,
+// so keep the number of server workers small to avoid overhead
+pub const DEFAULT_IP_ECHO_SERVER_THREADS: usize = MINIMUM_IP_ECHO_SERVER_THREADS;
```

Comment:
you can use this api:
https://doc.rust-lang.org/std/num/struct.NonZero.html#method.new_unchecked
here so not to do
```
NonZeroUsize::new(...).expect(...)
```
throughout the code.

Reply:
Yeah, it crossed my mind to do this at one point (have used this in the past). Not sure why I didn't go with it, but agreed that your suggestion is better. Done in https://github.com/anza-xyz/agave/pull/354/commits/4ac1e048ab907fb5d542748f4a6f65b24a61ed26